sys/newlib_syscalls_default/syscalls.c

```diff
@@ -461,6 +463,23 @@ int _unlink_r(struct _reent *r, const char *path)
 }
 #endif /* MODULE_VFS */
 
+/**
```

Comment:
can you add these functions in a separate PR?

Reply:
It's in #9090

Reply:
ok!

Reply:
This is rebased